subkey/src/main.rs

```diff
@@ -182,22 +182,21 @@ fn execute<C: Crypto>(matches: clap::ArgMatches) where
 
 			println!("Using a genesis hash of {}", HexDisplay::from(&genesis_hash.as_ref()));
 
-			let raw_payload = (
+			let raw_payload = SignedPayload::from_raw(
```

Comment:
great that this wasidentified! I exactly had in on a personal todo list to: check subkey with signed extension. If not here, at some point I would like to add some tests to it to make sure subkey is always generating stuff that can be verified by `node` (if that's a correct assumption).